### PR TITLE
build: use whiskers

### PR DESCRIPTION
**Pull request by uncenter**
Adds [Whiskers](https://github.com/catppuccin/toolbox/tree/main/whiskers), Catppuccin's internal templating tool, to build the themes. Instead of editing the theme files directly, edit the `mako.tera` file and then run `just build` to build the output themes. You will need Whiskers (linked above) installed and *optionally* [Just](https://github.com/casey/just) (for the convenient `just build` command, though you can skip it and delete the `justfile` if you want). I've also slightly updated the usage instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,24 @@
 <h3 align="center">
 	<img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/logos/exports/1544x1544_circle.png" width="100" alt="Logo"/><br/>
 	<img src="https://raw.githubusercontent.com/catppuccin/catppuccin/main/assets/misc/transparent.png" height="30" width="0px"/>
-	Catppuccin for <a href="https://github.com/emersion/mako">Mako</a>
+	Catppuccin for <a href="https://github.com/emersion/mako">mako</a>
```
Comment (uncenter) [inline README.md]: Looks like this is (now?) the correct capitalization of the project.

File: justfile
Comment (uncenter) [inline justfile:1]: Again, totally optional. Lmk if you want me to remove! We use it around Catppuccin so it is nice for consistency and ease of use though :p

**Comment by aquifolly** [comment]
could we use accent colors for borders?

**Comment by uncenter** [comment]
> could we use accent colors for borders?

This PR doesn't change any of the actual output themes, just how they are built. That could be changed after this is merged.

**Review by sgoudham — APPROVED** [review]
@lokesh-krishna Are you happy if I merge this? It looks good to me :+1:

**Comment by lokesh-krishna** [comment]
Looks good to me too!

**Pull request merged**